java/spacewalk-java.changes.rjpmestre.fix_proxy_config_fqdn_mapping

```diff
@@ -0,0 +1,2 @@
+- fix FQDN machine name mapping on proxy configuration
+- fix proxy configuration error display
```

Comment:
This is for users to read: you don't have to be too specific. Think about you would like to see here as a user that indicates your bug is fixed. And I wouldn't have 2 entries here: one is enough

Reply:
Got you! To see it as end user is a good hint.